import/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/CFindServiceTest.java

```diff
@@ -117,6 +120,28 @@ public JSONArray qidoRs(String path) throws DicomWebException {
       }
     }, Status.NotAuthorized);
   }
+  
+  @Test
+  public void testCFindService_withFuzzyMatching() throws Exception {
+	cFINDFlags.fuzzyMatching = true;
+    basicCFindServiceTest(new TestUtils.DicomWebClientTestBase() {
+      @Override
+      public JSONArray qidoRs(String path) throws DicomWebException {
+        throw new DicomWebException("test-generated exception", Status.InvalidAttributeValue);
+      }
+    }, Status.InvalidAttributeValue);
```

Comment:
Since this isn't testing an error case, I think we should return a dummy response and assert for a success status in both of these tests. E.g:

 @Override
      public JSONArray qidoRs(String path) throws DicomWebException {
        JSONArray instances = new JSONArray();
        instances.put(TestUtils.dummyQidorsInstance());
        return instances;
      }
    }, Status.Success);

Reply:
done. I also double checked the contributor license agreement. I can find my Individual Agreements on file that I signed on Jul 02, 2020.